cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/TemporalTest.scala

```diff
@@ -104,7 +104,7 @@ class TemporalTest extends DocumentingTest {
           |* Specifying both the offset and the time zone name (with the requirement that these match)
         """.stripMargin)
       p(
-        """The named time zone form uses the time zone rules of the time zone database to manage _daylight savings time_ (DST).""".stripMargin)
+        """The named time zone form uses the rules of the IANA time zone database to manage _daylight savings time_ (DST).""".stripMargin)
```

Comment:
We base our rules on the JVM rules which in turn are based on IANA. But I do not believe we directly reflect IANA at all times, which is what this statement seems to imply. Perhaps check with Satia on the exact rules here.

Reply:
The only thing that we support in addition to IANA is deleted timezone. That is not worth mentioning here. Other than that, we exactly use the IANA rules.